ui/src/map/tools/select-tool.ts

```diff
@@ -8,7 +8,18 @@ export const selectTool: MapTool = {
         const clickEvent = map.on('click', ({ coordinate }) => {
             const hitArea = getDefaultHitArea(map, coordinate);
             const items = searchItemsFromLayers(hitArea, layers, { limit: 1 });
-            options.onSelect(items);
+
+            options.onSelect({
+                ...items,
```

Comment:
Toimisikos tämä niin päin että laittaa ilman ?? Vain tyhjät taulukot ensin ja tuon ...items listan loppuun? Tyhjälle valintajoukolle taisi myös olla joku valmis init-olio määriteltynä joten voisi laittaa vain {...emptySelection, ...items}